pallets/kensetsu/src/migrations.rs

```diff
@@ -126,13 +127,19 @@ pub mod v1_to_v2 {
             pub interest_coefficient: FixedU128,
         }
 
-        impl<Moment> CollateralInfo<Moment> {
+        impl<Moment: AtLeast32Bit> CollateralInfo<Moment> {
             pub fn into_v2(self) -> crate::CollateralInfo<Moment> {
+                let mut new_risk_parameters = self.risk_parameters;
+                // It is rough approximation, but it is fast. Need to reset parameters after the
+                // migration.
+                new_risk_parameters.stability_fee_rate = new_risk_parameters
+                    .stability_fee_rate
+                    .saturating_mul(FixedU128::from_u32(1000));
                 crate::CollateralInfo {
-                    risk_parameters: self.risk_parameters,
+                    risk_parameters: new_risk_parameters,
                     total_collateral: self.total_collateral,
                     stablecoin_supply: self.kusd_supply,
-                    last_fee_update_time: self.last_fee_update_time,
+                    last_fee_update_time: self.last_fee_update_time / 1000u32.into(),
```

Comment:
Also should be checked

Reply:
No, there is no way to recover from error in migration.

Reply:
At least log the error

Reply:
Please, just log the error